packages/components/src/Iframe/styles.ts

```diff
@@ -0,0 +1,17 @@
+import { styled } from '../styles';
+
+export const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+`;
+
+export const IframeStyled = styled.iframe`
```

Comment:
```suggestion
export const StyledIframe = styled.iframe`
```

Reply:
https://kaluga-astral.github.io/style-guide/docs/rules/react/styled#%D0%B4%D0%BE%D0%BF%D1%83%D1%81%D0%BA%D0%B0%D0%B5%D1%82%D1%81%D1%8F-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5-styled-%D0%BF%D1%80%D0%B5%D1%84%D0%B8%D0%BA%D1%81%D0%B0-%D0%B4%D0%BB%D1%8F-%D0%BE%D0%B1%D0%B5%D1%80%D1%82%D0%BE%D0%BA-styled-%D0%BA%D0%BE%D0%BC%D0%BF%D0%BE%D0%BD%D0%B5%D0%BD%D1%82%D0%BE%D0%B2